src/components/HomePage/AccountHeader/AccountHeader.container.ts

```diff
@@ -1,26 +1,26 @@
-import { RootState } from '../../modules/reducer'
+import { RootState } from '../../../modules/reducer'
 import { getData as getProfiles } from 'decentraland-dapps/dist/modules/profile/selectors'
 import {
   MapStateProps,
   MapDispatchProps,
-  MapDispatch,
-} from '../HomePage/AccountHeader/AccountHeader.types'
+  MapDispatch
+} from './AccountHeader.types'
 import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
 import { connect } from 'react-redux'
-import { AccountHeader } from '../HomePage/AccountHeader'
+import AccountHeader from './AccountHeader'
 
 const mapState = (state: RootState): MapStateProps => {
   const address = getAddress(state)
   const profiles = getProfiles(state)
   const profile = address && address in profiles ? profiles[address] : null
```

Comment:
Can we use a selector to get the profile?

Reply:
totally agree with this!